doc/releases/migration-guide-3.7.rst

```diff
@@ -116,6 +116,11 @@ Bluetooth
 Bluetooth Mesh
 ==============
 
+* The model metadata pointer declaration of :c:struct:`bt_mesh_model` has been changed
+  to add `const` qualifiers. The data pointer of :c:struct:`bt_mesh_models_metadata_entry`
+  got `const` qualifier too. The model's metadata structure and metadata raw value
```

Comment:
```suggestion
  to add ``const`` qualifiers. The data pointer of :c:struct:`bt_mesh_models_metadata_entry`
  got ``const`` qualifier too. The model's metadata structure and metadata raw value
```

Reply:
Hi @kartben, could you take a look at the changes? Are they Ok now?